lib/ruby_lsp/requests/support/rubocop_diagnostic.rb

```diff
@@ -88,6 +100,79 @@ def offense_replacements
             )
           end
         end
+
+        sig { returns(Interface::CodeAction) }
+        def disable_line_action
+          Interface::CodeAction.new(
+            title: "Disable #{@offense.cop_name} for this line",
+            kind: Constant::CodeActionKind::QUICK_FIX,
+            edit: Interface::WorkspaceEdit.new(
+              document_changes: [
+                Interface::TextDocumentEdit.new(
+                  text_document: Interface::OptionalVersionedTextDocumentIdentifier.new(
+                    uri: @uri.to_s,
+                    version: nil,
+                  ),
+                  edits: line_disable_comment,
+                ),
+              ],
+            ),
+          )
+        end
+
+        sig { returns(T::Array[Interface::TextEdit]) }
+        def line_disable_comment
+          eol = Interface::Position.new(
+            line: @offense.line - 1,
+            # TODO: what's the real way to do this?
+            character: @offense.source_line.bytes.length,
```

Comment:
Finds the end of the line, but probably not the best way to do it.

`Offense#source_line` appears to be `@private`. Not sure if `bytes.length` is correct (but seems to work.)

Also doesn't handle multiline quotes, etc.